tests/test_dpmjetIII.py

```diff
@@ -39,3 +50,35 @@ def run_cross_section_ntrials():
 
 def test_cross_section_ntrials():
     run_in_separate_process(run_cross_section_ntrials)
+
+
+def test_cross_section_pp():
+    c = run_in_separate_process(run_cross_section, "p", "p")
+    assert_allclose(c.total, 38.9, atol=0.1)
+    assert_allclose(c.inelastic, 32.1, atol=0.1)
+    assert_allclose(c.elastic, 6.9, atol=0.1)
+    assert_allclose(
```

Comment:
If 38.9, 32.1 and 6.9 are some kind of constants, maybe it is better to give them names like `total_at_10GeV` and so on.

Reply:
Or leave a comment that those numbers are for specific kinematics

Reply:
I addeed a comment, but it's just trivial. Simply what DPMJET returns for this calls today and not related to any data reference.